build_tools/bazel/glob_lit_test.bzl

```diff
@@ -0,0 +1,116 @@
+# Test definitions for Lit, the LLVM test runner.
```

Comment:
Let's add a license header calling out that this file comes from TensorFlow?

Reply:
Done.

Reply:
Done.